notifier.rst

```diff
@@ -92,6 +92,7 @@ Service             Package                                DSN
 `Sinch`_            ``symfony/sinch-notifier``             ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
 `Smsapi`_           ``symfony/smsapi-notifier``            ``smsapi://TOKEN@default?from=FROM``
 `SmsBiuras`_        ``symfony/sms-biuras-notifier``        ``smsbiuras://UID:API_KEY@default?from=FROM&test_mode=0``
+`Smsbox`_           ``symfony/smsbox-notifier``            ``smsbox://APIKEY@default?mode=MODE&strategy=STRATEGY&sender=SENDER``
```

Comment:
Link declaration must be add at end of file

After
https://github.com/symfony/symfony-docs/blob/9ccbd0c7b572c2bf1b64fe4e9109566376ecba3a/notifier.rst?plain=1#L1003

Reply:
Done in fc23fa362